onebusaway-android/src/main/java/org/onebusaway/android/ui/SituationDialogFragment.java

```diff
@@ -162,10 +162,12 @@ public void onClick(DialogInterface dialog, int which) {
         TextView title = (TextView) dialog.findViewById(R.id.alert_title);
         title.setText(args.getString(TITLE));
 
-        TextView desc = (TextView) dialog.findViewById(R.id.alert_description);
+        TextView descTxtView = (TextView) dialog.findViewById(R.id.alert_description);
 
-        if (desc != null) {
-            desc.setText(Html.fromHtml(args.getString(DESCRIPTION)));
+        String desc = args.getString(DESCRIPTION);
+
+        if (descTxtView != null &&!TextUtils.isEmpty(desc)) {
```

Comment:
please be sure to watch your formatting on lines like this. I'd prefer to see this as:

```
if (descTxtView != null && !TextUtils.isEmpty(desc)) {
```

Reply:
Thank you for correcting me, it's more clear now!